### PR TITLE
Update to consistent formatting 

### PR DESCRIPTION
**Pull request by sarahwooders**
* Updated `poetry.lock` 
* Ran `poetry run black . -l 140` 
* Updated README.md

### CODE REVIEW

**Comment by cpacker** [comment]
[😳 holay molay](https://www.youtube.com/shorts/YxR7xOUKt9s)

File: README.md [hunk 1]
```diff
@@ -345,3 +345,17 @@ Datasets used in our [paper](https://arxiv.org/abs/2310.08560) can be downloaded
...
+
+## Development 
+You can install MemGPT from source with: 
+```
+git clone git@github.com:cpacker/MemGPT.git
+poetry shell
+poetry install
+```
+
+### Contributing
```
Comment (vivi) [inline README.md:357]: stick this in CONTRIBUTING.md

**Review by vivi — COMMENTED** [review]
we should also set up a commit hook

**Comment by cpacker** [comment]
@vivi chad tells me to update `settings.json` with:

```json
{
    "[python]": {
        "editor.defaultFormatter": "ms-python.black-formatter",
        "editor.formatOnSave": true,
        "python.formatting.blackArgs": [
            "--line-length",
            "140"
        ],
        "python.formatting.provider": "black"
    }
}
```

prev:

```json
    "[python]": {
        "editor.defaultFormatter": "ms-python.black-formatter",
        "editor.formatOnSave": true
    }
```

**Comment by vivi** [comment]
@cpacker Maybe just
```
    "black-formatter.args": ["-l", "140"],
```

**Comment by cpacker** [comment]
ChatGPT strikes and misses yet again, I should have chatted with the Black Formatter docs via MemGPT instead smh.

**Review by cpacker — APPROVED** [review]
lgtm 😈

**Pull request merged**